util/subtree_linux.go

```diff
@@ -2,13 +2,21 @@ package util
 
 import (
 	"bytes"
+	"context"
 	"fmt"
 	"io/ioutil"
 	"os"
+	"time"
 
 	"github.com/mongodb/grip"
 )
 
+const (
+	processCleanupAttempts   = 10
+	processCleanupTimeoutMin = 1 * time.Second
```

Comment:
This will end up being many minutes, which I think is too long for Evergreen. If a host is idle for this period of time, for example, Evergreen will reap it, so we need to stay far underneath that threshold. This also increases latency between tasks, so I think it's valuable to keep this short. I also think that if a process stays around for even 1 minute, that's far too long, and we should log it.

I think we should change the minimum to 100ms. I'm less sure about the max.  If set to 1 second, the total time will be about 6.5 seconds. That seems maybe a little bit too quick, but maybe that's okay for a first try at this, especially because the cleanup code is called a few times in the course of task execution.